_posts/admin/2020-04-17-enable-mptv2.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2020-04-17
+title: Enabling the Managed Pipeline Templates UI
+categories:
+   - Admin
+description: How to enable the Managed Pipeline Templates UI in Armory Spinnaker
+type: Document
+---
+
+Armory Spinnaker 2.19 contains the latest version of Managed Pipeline Templates, which is the default pipeline templating solution offered in OSS Spinnaker. 
+
+Instead, Armory recommends using Armory's Pipeline as Code feature because it offers the following benefits:
```

Comment:
After moving the other stuff down "Instead" doesn't feel right?

However it is recommended using Armory's Pipeline as Code feature because it offers the following benefits:

Feel free to ignore if "Instead" sounds better to you.